### PR TITLE
Updated versions in readme

### PR DESCRIPTION
**Pull request by soeren-schmaljohann-2denker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Workflows used at [ZWEIDENKER](https://zweidenker.de) to build and deploy Flutte
 
 ```yaml
   quality:
-    uses: zweidenker/flutter_workflows/.github/workflows/melos_quality_checks.yaml@v1
+    uses: zweidenker/flutter_workflows/.github/workflows/melos_quality_checks.yaml@v1.0.1
     with:
         setup: true
         lint: false
```
Comment (ABausG) [inline README.md:12]:
```suggestion
        lint: false
        test: true
```

Could you add the missing space in test? :D

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Workflows used at [ZWEIDENKER](https://zweidenker.de) to build and deploy Flutte
 
 ```yaml
   quality:
-    uses: zweidenker/flutter_workflows/.github/workflows/melos_quality_checks.yaml@v1
+    uses: zweidenker/flutter_workflows/.github/workflows/melos_quality_checks.yaml@v1.0.1
```
Comment (ABausG) [inline README.md:9]: How about we just say: Latest Version here and manually check? That way we don't need to adjust this for each release we do...

Comment (soeren-schmaljohann-2denker) [reply README.md:9]: We could, but I feel like that would still lead to the same problem of copy'n'pasting the code, running it once and only then realizing, that you had to replace/update something in it.

File: README.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ Workflows used at [ZWEIDENKER](https://zweidenker.de) to build and deploy Flutte
     with:
         setup: true
         lint: false
+        test: true
         test:true
```
Comment (ABausG) [inline README.md]: Now there is one line too much :D

Comment (soeren-schmaljohann-2denker) [reply README.md]: done

**Review by ABausG — APPROVED** [review]

**Pull request merged**